README.md

```diff
@@ -76,52 +76,61 @@ python -m pip install pennylane
 
 ## Getting started
 
-For an introduction to quantum machine learning, guides and resources are available on
-PennyLane's [quantum machine learning hub](https://pennylane.ai/qml/):
+Get up and running quickly with PennyLane by following our [quickstart guide](https://docs.pennylane.ai/en/stable/introduction/pennylane.html), designed to introduce key features and help you start building quantum circuits right away.
+
+Whether you're exploring quantum machine learning (QML), quantum computing, or quantum chemistry, PennyLane offers a wide range of tools and resources to support your research:
 
 <img src="https://raw.githubusercontent.com/PennyLaneAI/pennylane/master/doc/_static/readme/gpu_to_qpu.png" align="right" width="400px">
 
-* [What is quantum machine learning?](https://pennylane.ai/qml/whatisqml)
-* [QML tutorials and demos](https://pennylane.ai/qml/demonstrations)
-* [Frequently asked questions](https://pennylane.ai/faq)
-* [Key concepts of QML](https://pennylane.ai/qml/glossary)
-* [QML videos](https://pennylane.ai/qml/videos)
+### Key Resources:
+
+* [Learn quantum programming](https://pennylane.ai/qml/)
+* [PennyLane Codebook](https://pennylane.ai/codebook/)
+* [Coding challenges](https://pennylane.ai/challenges/)
+* [Demos](https://pennylane.ai/qml/demonstrations.html)
+* [Frequently asked questions](https://pennylane.ai/faq.html)
+* [Key concepts](https://pennylane.ai/qml/glossary.html)
+* [Videos](https://pennylane.ai/qml/videos.html)
+
 
 You can also check out our [documentation](https://pennylane.readthedocs.io) for [quickstart
 guides](https://pennylane.readthedocs.io/en/stable/introduction/pennylane.html) to using PennyLane,
 and detailed developer guides on [how to write your
 own](https://pennylane.readthedocs.io/en/stable/development/plugins.html) PennyLane-compatible
 quantum device.
 
-## Tutorials and demonstrations
+## Demos
 
-Take a deeper dive into quantum machine learning by exploring cutting-edge algorithms on our [demonstrations
-page](https://pennylane.ai/qml/demonstrations).
+Take a deeper dive into quantum computing by exploring cutting-edge algorithms using PennyLane and quantum hardware. [Explore PennyLane demos](https://pennylane.ai/qml/demonstrations.html).
 
 <a href="https://pennylane.ai/qml/demonstrations">
   <img src="https://raw.githubusercontent.com/PennyLaneAI/pennylane/master/doc/_static/readme/demos.png" width="900px">
 </a>
 
-All demonstrations are fully executable, and can be downloaded as Jupyter notebooks and Python
-scripts.
-
 If you would like to contribute your own demo, see our [demo submission
 guide](https://pennylane.ai/qml/demos_submission).
 
-## Videos
+## Research Applications
+
+PennyLane is at the forefront of research in quantum computing, quantum machine learning, and quantum chemistry. Discover how researchers are using PennyLane.

```

Comment:
@justinpickering one suggestion for this part of the readme is to simplify it, and not explicitly link papers here, but simply link out to the research page (since the readme will be harder to keep up to date as frequently).